thumbor/loaders/__init__.py

```diff
@@ -10,11 +10,6 @@
 
 
 class LoaderResult(object):
-
-    ERROR_NOT_FOUND = 'not_found'
```

Comment:
@gi11es Removing those const would be backward incompatible. For thumbor 6 it should be possible to support both constants and response codes.

```
                elif result.loader_error == LoaderResult.ERROR_TIMEOUT:
                     # Return a Gateway Timeout status if upstream timed out (i.e. 599)
                     self._error(504)
                 elif isinstance(result.loader_error, int):
                     self._error(result.loader_error)
                      return
```